src/main/java/Pebble/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package pebble;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Subclass of Tasks that has a deadline.
+ */
+public class Deadline extends Task {
+    /** When deadline date is properly formatted */
+    protected LocalDate by;
+    /** When deadline date cannot be formatted from string will be left as string */
+    protected String stringBy;
+
+    /**
+     *  Constructor for Deadline.
```

Comment:
Comments for methods should start with verbs, not sure if its the case for constructors?